Networking/Networking/Model/Stats/OrderStatsV4Interval.swift

```diff
@@ -0,0 +1,60 @@
+/// Represents an single order stat for a specific period.
+/// v4 API
+public struct OrderStatsV4Interval: Decodable {
+    public let interval: String
+    public let dateStart: String
+    public let dateEnd: String
```

Comment:
I also considered that. In the end I decided to go with a string instead of a date because, being dates as delicate as they are to deal with, it might be necessary to do some logic around them (i.e. consider the system timezone, or offset them somehow) and that might make sense to be done in the business logic layer (or as you mentioned, the view model).

Maybe we could revisit this decision after having the full implementation in place? If we notice that we'd be better off just having instances of Date here, we can alway come back to it and change it?

Reply:
Yep! It's not blocking anything and I really don't have an opinion. I thought I should bring it up since we were on the topic.